src/test/java/eu/europeana/metis/sandbox/controller/DatasetControllerIT.java

```diff
@@ -27,18 +32,32 @@
   private MockMvc mvc;
 
```

Comment:
There are more calls to test here

Reply:
correct, no more calls

Reply:
Just realized, the OAI-PMH test call is not here, you don't want to add that one?